Spark/metrics.csv

```diff
@@ -0,0 +1,207 @@
+spark.executor.failed_tasks.max,gauge,,task,,The max number of failed tasks in the application's executors,0,spark,exe fld tsk mx
+spark.executor.failed_tasks.avg,gauge,,task,,The average number of failed tasks in the application's executors,0,spark,exe fld tsk avg
+spark.executor.failed_tasks.median,gauge,,task,,The median number of failed tasks in the application's executors,0,spark,exe fld tsk med
+spark.executor.failed_tasks.95percentile,gauge,,task,,The 95th percentile number of failed tasks in the application's executors,0,spark,exe fld tsk 95
+spark.executor.failed_tasks.count,rate,,task,,The number of times the number of failed tasks in the application's executors was sampled,0,spark,exe fld tsk cnt
+spark.executor.completed_tasks.max,gauge,,task,,The max number of completed tasks in the application's executors,0,spark,exe comp tsk mx
+spark.executor.completed_tasks.avg,gauge,,task,,The average number of completed tasks in the application's executors,0,spark,exe comp tsk avg
+spark.executor.completed_tasks.median,gauge,,task,,The median number of completed tasks in the application's executors,0,spark,exe comp tsk med
+spark.executor.completed_tasks.95percentile,gauge,,task,,The 95th percentile number of completed tasks in the application's executors,0,spark,exe comp tsk 95
+spark.executor.completed_tasks.count,rate,,task,,The number of times the number of completed tasks in the application's executors was sampled,0,spark,exe comp tsk cnt
+spark.executor.total_tasks.max,gauge,,task,,The max total number of tasks in the application's executors,0,spark,exe tot tsk mx
+spark.executor.total_tasks.avg,gauge,,task,,The average total number of tasks in the application's executors,0,spark,exe tot tsk avg
+spark.executor.total_tasks.median,gauge,,task,,The median total number of tasks in the application's executors,0,spark,exe tot tsk med
+spark.executor.total_tasks.95percentile,gauge,,task,,The 95th percentile total number of tasks in the application's executors,0,spark,exe tot tsk 95
+spark.executor.total_tasks.count,rate,,task,,The number of times the total number of tasks in the application's executors was sampled,0,spark,exe tot tsk cnt
+spark.executor.total_duration.max,gauge,,millisecond,,The max total duration of the application's executors,0,spark,exe tot dur mx
+spark.executor.total_duration.avg,gauge,,millisecond,,The average total duration of the application's executors,0,spark,exe tot dur avg
+spark.executor.total_duration.median,gauge,,millisecond,,The median total duration of the application's executors,0,spark,exe tot dur med
+spark.executor.total_duration.95percentile,gauge,,millisecond,,The 95th percentile total duration of the application's executors,0,spark,exe tot dur 95
+spark.executor.total_duration.count,rate,,millisecond,,The number of times the total duration of the application's executors was sampled,0,spark,exe tot dur cnt

```

Comment:
For `spark.executor.total_duration` let's specify that this is the amount of time executors spent executing tasks, right now it can be mistaken for the duration that executors have been running.